tests/test_github_bookmarks.py

```diff
@@ -161,31 +150,34 @@ def test_run(self):
                     self.assertIsNotNone(second_bookmark_key_value.get('since'))
 
                     # Verify the second sync bookmark is Equal or Greater than the first sync bookmark
-                    # the tap uses `time_extracted` and sets a bookmark using `since` for all real/pseudo incremental streams
-                    self.assertGreaterEqual(second_bookmark_value, first_bookmark_value)
-
-                    for record in second_sync_messages:
-                        # Verify the second sync bookmark value is the max replication key value for a given stream
-                        if stream in ('commits', 'pr_commits'):
-                            replication_key_value = record.get('commit', {}).get('committer', {}).get('date')
-                        else:
-                            replication_key_value = record.get(replication_key)
-                        self.assertLessEqual(
-                            replication_key_value, second_bookmark_value_utc,
-                            msg="Second sync bookmark was set incorrectly, a record with a greater replication-key value was synced."
-                        )
+                    self.assertGreaterEqual(second_bookmark_value_ts, first_bookmark_value_ts)
 
+                    REPLICATION_KEY_FORMAT = self.RECORD_REPLICATION_KEY_FORMAT
```

Comment:
As it is not a constant, no need to define in all caps

Reply:
Updated